### ISSUE TITLE
Setup "Clone repository" uses git+ssh instead of the more common https://

### ISSUE THREAD

**Issue by zetlen** [magento-research/pwa-studio#168]
As we discovered in Slack, some people are following the Setup instructions up to the [Clone repository step](https://magento-research.github.io/pwa-studio/venia-pwa-concept/setup/#clone-repository), but they get an authentication error from GitHub when they run that command.

[![Example from Slack](https://user-images.githubusercontent.com/1643758/43079951-1c0edb28-8e54-11e8-882f-f73fd54d94ee.png)](https://magentocommeng.slack.com/archives/C71HNKYS2/p1532157375000037)

## This issue is for the following packages:

[ ] `venia-concept`
[ ] `pwa-buildpack`
[ ] `peregrine`
[ ] `pwa-module`
[x] `pwa-devdocs`

## This issue is a:

[ ] Bug
[ ] Feature suggestion
[x] Documentation issue
[ ] Other (Please Specify)

### Environment

Any Git client

## Description

A new user copies and pastes the shell command from the [Clone repository step](https://magento-research.github.io/pwa-studio/venia-pwa-concept/setup/#clone-repository):
```sh
git clone git@github.com:magento-research/pwa-studio.git
```

### Expected result:

The repository clones into a local directory

### Actual result:

GitHub refuses the connection because it can't make public key authentication

## Possible solutions:

**TLDR Replace that line with `git clone https://github.com/magento-research/pwa-studio.git`.** We should just use HTTPS URLs everywhere per GitHub's recommendation. 

Most users don't use the git+ssh protocol that URL requires. If the user has never set up SSH with GitHub, they'll see an auth error.

Background: [Git provides two transport options](https://help.github.com/articles/which-remote-url-should-i-use/), and for some reason, [some people](https://github.com/zetlen) choose to use the less popular and less supported one. Anywhere we find one of those git@github.com URLS in our docs, we should replace it with the HTTPS url that most people use.

**Comment by mhhansen** [comment]
I think this one can be closed [PR#170](https://github.com/magento-research/pwa-studio/pull/170) has been merged

**Comment by zetlen** [comment]
Oh indeed! Closed by #170.
